File: frontend/src/features/actividades/corregir_reparo_control_credito/models/corregirReparoControlCredito.ts
export interface CorregirReparoControlCredito {
  id_corregir_reparo_control_credito?: number | null;
  id_realizar_control_credito?: number | null;
  id_expediente?: number | null;
  id_usuario_solicitante?: number | null;
  subsanar?: boolean | null;
  observaciones?: string | null;
  
  id_solicitud?: number | null;
  id_solicitante?: number | null;
  solicitante?: string | null;
  observaciones_reparo?: string | null;
  fecha_ingreso?: string | null;

  is_active?: boolean | null;
  row_status?: boolean | null;
  created_by?: number | null;
  created_date?: string | null;
  modified_by?: number | null;
  modified_date?: string | null;
}

File: backend/Multibanca.Domain.Models/Multibanca/corregir_reparo_control_credito.cs
using Common.Domain.Models;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Multibanca.Domain.Models.Multibanca
{
    public class corregir_reparo_control_credito : base_auditoria
    {
        public int id_corregir_reparo_control_credito { get; set; }
        public int id_realizar_control_credito { get; set; }
        public long id_expediente { get; set; }
        public int id_usuario_solicitante { get; set; }
        public bool subsanar { get; set; }
        public string? observaciones { get; set; }
        public int id_solicitud {  get; set; } 
        public int id_solicitante { get; set; }
        public string? solicitante { get; set; }
        public string? observaciones_reparo { get; set; }
        public DateTime? fecha_ingreso { get; set; }
    }
}
